flux_led/__init__.py

```diff
@@ -450,14 +450,16 @@ def __init__(self, ipaddr, port=5577, timeout=5):
         self.port = port
         self.timeout = timeout
 
+        self.protocol = None
```

Comment:
How about making calling setProcotol() if protocol has been passed on the init() function?  Something like:

```python
def __init__(self, ipaddr, port=5577, timeout=5, protocol=None):

    if protocol:
        self.setProtocol(protocol)
```

This will be interesting for future use when not calling the command line but the library functions.